frontend/src/concepts/hardwareProfiles/HardwareProfileTableColumn.tsx

```diff
@@ -0,0 +1,90 @@
+import { HelperText, HelperTextItem, Spinner, Flex, FlexItem } from '@patternfly/react-core';
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ModelResourceType } from '@odh-dashboard/model-serving/extension-points';
+import { SupportedArea, useIsAreaAvailable } from '#~/concepts/areas';
+import { NotebookKind } from '#~/k8sTypes';
+import { ContainerResources } from '#~/types';
+import ScopedLabel from '#~/components/ScopedLabel';
+import { ScopedType } from '#~/pages/modelServing/screens/const';
+import { getHardwareProfileDisplayName } from '#~/pages/hardwareProfiles/utils';
+import { resourceTypeOf } from '#~/concepts/hardwareProfiles/utils';
+import { useHardwareProfileBindingState } from './useHardwareProfileBindingState';
+import HardwareProfileDetailsPopover from './HardwareProfileDetailsPopover';
+import HardwareProfileBindingStateLabel from './HardwareProfileBindingStateLabel';
+import { HardwareProfileBindingState } from './const';
+
+type HardwareProfileTableColumnProps = {
+  namespace: string;
+  resource: NotebookKind | ModelResourceType;
+  containerResources: ContainerResources | undefined;
+  isActive?: boolean;
+};
+
+const HardwareProfileTableColumn: React.FC<HardwareProfileTableColumnProps> = ({
+  namespace,
+  resource,
+  containerResources,
+  isActive = false,
+}) => {
+  const isProjectScoped = useIsAreaAvailable(SupportedArea.DS_PROJECT_SCOPED).status;
+  const [bindingStateInfo, bindingStateLoaded, loadError] =
+    useHardwareProfileBindingState(resource);
+
+  const hardwareProfile = bindingStateInfo?.profile;
+
+  if (loadError && bindingStateInfo?.state !== HardwareProfileBindingState.DELETED) {
+    return (
+      <HelperText>
+        <HelperTextItem variant="error">Custom</HelperTextItem>
```

Comment:
This is not custom its just an error. I would put a error icon and then a popover to show the load error for the content of the popover. a popover can have alertSeverityVariant you can set to error

Reply:
Updated